Sources/SymbolKit/UnifiedSymbolGraph/UnifiedSymbol.swift

```diff
@@ -12,7 +12,7 @@ import Foundation
 
 extension UnifiedSymbolGraph {
     /// A combined symbol from multiple languages' views of the same symbol.
-    public class Symbol {
+    public struct Symbol {
```

Comment:
What's the reason for making this a struct? It's a breaking change for any code that relies on the unified symbol being a class.

Reply:
I think @franklinsch called this out. The original `SymbolGraph.Symbol` type is a struct, so i think this is mainly meant to bring that into parity.

Reply:
I think this particular change could be addressed in a separate PR to avoid rocking the boat too much here. But we can keep the new `Relationship` type a struct.